DataFormats/RecoCandidate/interface/IsoDeposit.h

```diff
@@ -299,8 +302,10 @@ namespace reco {
     float theCandTag;
 
     //! the deposits identifed by relative position to center of cone and deposit value
-
     DepositsMultimap theDeposits;
+
+    //! RNTuple-safe persistent storage
+    std::vector<std::pair<Distance, float>> depositsPersistent;
```

Comment:
I believe I have what should be a working version of IsoDeposit. How about reverting this commit?

Reply:
Yeah I'll rebase this whole PR once I solve the `free` issue, and remove this migration.